docs/api/checkouts.rst

```diff
@@ -238,185 +373,291 @@ Response will contain an updated order information.
-                        "additional_item_cost": 500,
-                        "additional_item_limit": 10,
-                        "first_item_cost": 4000
-                    },
-                    "created_at": "2014-12-03T09:54:28+0000",
-                    "enabled": true,
-                    "id": 4,
-                    "name": "FedEx World Shipping",
-                    "updated_at": "2014-12-03T09:54:28+0000"
-                },
-                "state": "checkout",
-                "updated_at": "2014-12-15T14:30:41+0000"
+                ]
             }
-        ],
-        "shipping_address": {
-        },
-        "state": "cart",
-        "total": 1504750,
-        "updated_at": "2014-12-15T14:30:41+0000",
-        "user": {
-        }
+        ]
     }
 
+Response contains the proposed shipments and for each, it also has a list of shipping methods available with calculated price.
 
-Payment step
-------------
+.. warning::
 
-When we are done with shipping choices and we know the final price of an order, we can select a payment method.
+    Because of the custom calculation logic for this endpoint the regular rules of overriding does not apply for this endpoint. In order to achieve different message one has to provide custom controller and build the message by his own. Exemplary implementation can be found `here <https://github.com/Sylius/Sylius/blob/master/src/Sylius/Bundle/ApiBundle/Controller/ShowAvailableShippingMethodsController.php>`__
+
+Next step is updating the order with the types of shipping methods that one has selected. A PUT request has to be send for each available shipment.
 
-To obtain a list of available payment methods for this order, simply call a GET request again:
+Definition
+^^^^^^^^^^
 
 .. code-block:: text
 
-    GET /api/v1/checkouts/44
+    PUT /api/v1/checkouts/select-shipping/{id}
+
++------------------------+----------------+-------------------------------------------------------------------------------------------+
+| Parameter              | Parameter type | Description                                                                               |
++========================+================+===========================================================================================+
+| Authorization          | header         | Token received during authentication                                                      |
++------------------------+----------------+-------------------------------------------------------------------------------------------+
+| id                     | url attribute  | Id of the requested cart                                                                  |
++------------------------+----------------+-------------------------------------------------------------------------------------------+
+| shipments[X]['method'] | request        | Code of chosen shipping method (Where X is the number of shipment in the returned array.) |
```

Comment:
Code of **the** chosen shipping method (where X is a number od the shipment in the returned array)

`.` no dot in the brackets please.